index-2.html

```diff
@@ -332,7 +333,7 @@ <h3>Subscribe to our Newsletter for latest news.</h3>
 			<div class="row ">
 				<div class="col-md-4 col-sm-6">
 					<img src="img/logo-footer.png" width="190" height="48" alt="USIVA" data-retina="true" id="logo_footer">
-					<p>Id pri consul aeterno petentium. Vivendo abhorreant et vim, et quot persecuti mel. Libris hendrerit ex sea. Duo legere evertitur an, pri hinc doctus definitiones an, vix id dicam putent. Ius ornatus instructior in.</p>
+					<p>USIVA is dedicated to empowering you with expert immigration advice and support, ensuring your journey to the United States is smooth and successful.</p>
```

Comment:
@AbdulMoeez6 you removed more than 10% on this. Make sure it is 10% char changes only.